lib/ansible/plugins/callback/__init__.py

```diff
@@ -259,6 +259,11 @@ def _clean_results(self, result, task_name):
                 for hidme in self._hide_in_debug:
                     result.pop(hidme, None)
 
+    def _print_task_path(self, task):
+        path = task.get_path()
+        if path:
+            self._display.display(u"task path: %s" % path, color=C.COLOR_DEBUG)
```

Comment:
you probalby want same color as origin, default to debug

Reply:
Should that be `C.COLOR_ERROR` or `None` when verbosity < 2 called from `v2_runner_on_failed()`? I think it would be odd to have it be different colors depending on verbosity level, which is why I just kept it as `C.COLOR_DEBUG`.

Reply:
well, you might be displaying it in diff levels, so probably that should be the configuable part: `(self, task, display=v|error|warning)`